commit/merkleroot/outcome.go

```diff
@@ -125,12 +127,23 @@ func buildReport(
 
 	sort.Slice(roots, func(i, j int) bool { return roots[i].ChainSel < roots[j].ChainSel })
 
-	// TODO: use q.RMNSignatures in the generated outcome and eventually report. - Blocked by onchain work.
+	sigs := make([]cciptypes.RMNECDSASignature, 0)
+	if q.RMNSignatures == nil {
+		lggr.Error("RMNSignatures are nil")
+	} else {
+		parsedSigs, err := rmn.NewECDSASigsFromPB(q.RMNSignatures.Signatures)
+		if err != nil {
+			lggr.Errorw("Failed to parse RMN signatures", "error", err)
```

Comment:
q: if all nodes fail to parse sigs, outcome will have an empty sigs array, but what's going to happen in that case? No commit report posted onchain? Or will the commit report just fail to verify onchain because RMN sigs are empty?